lms/djangoapps/onboarding/models.py

```diff
@@ -901,3 +901,51 @@ class RegistrationType(models.Model):
         related_name='registration_type'
     )
     choice = models.SmallIntegerField(default=1, null=False)
+
+
+class Education(models.Model):
+    """
+    Model to store user education information
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True, unique=True)
+    user = models.ForeignKey(User, related_name='education',
+                             on_delete=models.CASCADE)
+    school_name = models.CharField(max_length=100, null=True, blank=True)
+    degree_name = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
Please change max_length to 255
Because CharField creates varchar field in database which will consume capacity according to actual characters means if school_name or degree_name is of 70 or 80 character then it will not take memory more then this

Please make this change in other models as well according to need

Reply:
done